addons/gamification/models/res_users.py

```diff
@@ -1,9 +1,17 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+import datetime
+
 from odoo import _, api, fields, models
 
 
+class ResUsersLog(models.Model):
+    _inherit = 'res.users.log'
+
+    last_interacted_date = fields.Datetime('Last interacted', default=lambda _: datetime.datetime.now())
```

Comment:
you shadow the `_` used for translation
```suggestion
    last_interacted_date = fields.Datetime('Last interacted', default=lambda __: datetime.datetime.now())
```